client/lcd/swagger-ui/swagger.yaml

```diff
@@ -30,28 +36,17 @@ paths:
     get:
       summary: Version of Gaia-lite
       tags:
-        - version
+        - Misc
       description: Get the version of gaia-lite running locally to compare against expected
       responses:
         200:
           description: Plaintext version i.e. "v0.25.0"
-  /node_version:
```

Comment:
not sure if this was the right one to remove

Reply:
yes, that's the one